Version 0.0.5 will not install on a clean windows install due to "latest" on yo dependency

I am unable to install version 0.0.5 on a windows system and a linux system with a clean node install. I get the following error :

npm ERR! peerinvalid Peer generator-mongoose@0.0.5 wants yo@latest

I can however install version 0.0.4 with out issue. I've made sure that I have the latest version of yo installed (1.1.2)

I am going to do a pull request with the version of yo specified in the package.json instead of using "latest"

My recommended fix is

FROM
"latest"
to
">=1.1.1"

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "mocha": "~1.14.0"
   },
   "peerDependencies": {
-    "yo": "latest"
+    "yo": ">=1.1.1"
   },
   "engines": {
     "node": ">=0.8.0",

